htdocs/expedition/card.php

```diff
@@ -1576,7 +1576,7 @@
 					$srcLine = new OrderLine($db);
 					$srcLine->fetch_optionals($line->id); // fetch extrafields also available in orderline
 					$line = new ExpeditionLigne($db);
-					$line->fetch_optionals($line->id);
+					//$line->fetch_optionals($line->id);
 					$line->array_options = array_merge($line->array_options, $srcLine->array_options);
```

Comment:
If you dont't don the $line->fetch_optionals($line->id);
then the $line->array_options just after is not set